armada-ths/ais
# Cr logging

- Added logging for complete registration to help us keep track of what is going on. You can find it in admin/register. Each time the CR is saved or submitted, a new OrderLog should be created. 
- Hid save button when save button is present. 
- Hid message about technical issues on login page

To test: 

1. try to save and submit (at different times) and look in admin/register to see if the log showed up.
2.  click around the tabs and make sure save button is still there
3. go to confirm and submit. If accept Terms = True (the checkbox is checked) ONLY the submit button should be seen and save button hidden. If you check out the checkox submit and save should switch. 
4. Check the checkbox and make the submit button appear. Go to another tab: the save should replace submit.
5. Try to "break" the switch of the buttons ;)
